sanitizer/search_fallback.js

```diff
@@ -1,29 +0,0 @@
-var sanitizeAll = require('../sanitizer/sanitizeAll'),
```

Comment:
Was this file not used anywhere? If so, and it's not too much work, feel free to submit a separate PR to remove it. Many small PRs > one bigger one.